content/sensu-go/6.4/release-notes.md

```diff
@@ -85,6 +86,44 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 6.4.0 release notes
+
+**June 23, 2021** &mdash; The latest release of Sensu Go, version 6.4.0, is now available for download. 
+
+<!--// synopsis placeholder //-->
+
+See the [upgrade guide][1] to upgrade Sensu to version 6.4.0.
+
+**NEW FEATURES:**
+
+- ([Commercial feature][207]) The system information modal in the Web UI now includes license expiration information, accessed via the `CTRL + .` keyboard shortcut.
+- Added `darwin/arm64` to the list of platforms we build sensu-agent & sensuctl for.
```

Comment:
```suggestion
- Added binary-only distribution for [macOS arm64][].
```

Reply:
Add a link (bottom of this file) to `/sensu-go/6.4/platforms/#macos` to help people find this.